modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -1864,7 +1869,22 @@ else if (ttl != CU.TTL_ZERO)
                     false,
                     c.updateRes.updateCounter(),
                     fut,
-                    topVer);
+                    topVer,
+                    false);
+            } else if ((curLsnrs = cctx.continuousQueries().updateListeners(internal, false)) != null) {
+
+                cctx.continuousQueries().onEntryUpdated(curLsnrs,
+                    key,
+                    null,
```

Comment:
`newVal` and `oldVal` already null. 
Event would be skipped.
Why do we need to add the last parameter?

Reply:
fixed